orga/app/components/campaign/list.gjs

```diff
@@ -36,12 +36,12 @@ export default class List extends Component {
   @service notifications;
   @tracked showDeletionModal = false;
 
-  get listOnlyCampaignsOfCurrentUser() {
-    return this.args.listOnlyCampaignsOfCurrentUser ?? false;
+  get canDelete() {
+    return this.args.canDelete ?? false;
   }
 
   get caption() {
-    if (!this.listOnlyCampaignsOfCurrentUser) {
+    if (!this.canDelete) {
```

Comment:
**To do :** prendre la caption en paramètre (ici elle ne dépend pas de la possibilité de suppression)